tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyBulkClosePosts.java

```diff
@@ -110,10 +111,12 @@ private void updateList(ResultSet rs) throws SQLException {
     }
 
     private boolean matchesWinning(String loc, Integer xpath, String value) {
-        XMLSource diskData = sm.getDiskFactory().makeSource(loc).freeze();
+        STFactory stf = sm.getSTFactory();
+        XMLSource diskData = stf.makeSource(loc).freeze();
```

Comment:
This isn't diskData! This will give live (vetted) data 

Also  I think it's wrong to call freeze() on the STF's source, that will freeze out other callers—why?

Reply:
`sm.getDiskFactory().make(loc, true)` will give you a resolved CLDRFile

Reply:
the disk files ought to be be pre-frozen, not sure i they are or not